mlflow/models/evaluation/validation.py

```diff
@@ -0,0 +1,206 @@
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import BAD_REQUEST
+
+
+class MetricThreshold:
+    """
+    This class allows you to define metric thresholds for model validation.
+    Allowed thresholds are: threshold, min_absolute_change, min_relative_change.
+
+    :param threshold: A floating number representing the value threshold for the metric.
+
+                      - If higher is better for the metric, the metric value has to be
+                        >= threshold to pass validation.
+                      - Otherwise, the metric value has to be <= threshold to pass the validation.
+
+    :param min_absolute_change: A floating point number representing the minimum absolute change
+                                required for candidate model to pass the comparison with
+                                the baseline model.
+
+                                - If higher is better for the metric, metric value has to be
+                                  >= baseline model metric value + min_absolute_change
+                                  to pass the validation.
+                                - Otherwise, metric value has to be
+                                  <= baseline model metric value + min_absolute_change
+                                  to pass the validation.
+
+    :param min_relative_change: A floating point number between 0 and 1 representing
+                                the minimum relative change (in percentage of
+                                baseline model metric value) for candidate model
+                                to pass the comparison with the baseline model.
+
+                                - If higher is better for the metric, metric value has to be
+                                  >= baseline model metric value * (1 + min_relative_change)
+                                - Otherwise, metric value has to be
+                                  <= baseline model metric value * (1 - min_relative_change)
+
+    :param higher_is_better: A boolean representing whether higher value is better for the metric.
+    """
+
+    def __init__(
+        self,
+        threshold=None,
+        min_absolute_change=None,
+        min_relative_change=None,
+        higher_is_better=None,
```

Comment:
Seems like `higher_is_better` is a required parameter. In that case, can we move it to be before `threshold` and not give it any default value? Then, in the doc string, change the position of the parameter doc string.

Reply:
Another option though would be to just change the doc string to specify that the argument is required. @dbczumar what are your thoughts on this?

Reply:
Actually, let us just update the doc string to say required.